src/journeys_utils.js

```diff
@@ -460,6 +469,13 @@ journeys_utils.animateBannerEntrance = function(banner, cssIframeContainer) {
 				// check if safeAreaRequired is true or not
 				if (journeys_utils.journeyLinkData && journeys_utils.journeyLinkData['journey_link_data'] && !journeys_utils.journeyLinkData['journey_link_data']['safeAreaRequired']) {
 					banner.style.bottom = '0';
+
+					if (metadata.type === "center_overlay") {
+						banner.style.bottom = "140px";
+                        banner.style.width = "94%";
+                        banner.style.borderRadius = "20px";
```

Comment:
We should move this to separate function like the **else** case 

````
else {
					journeys_utils._dynamicallyRepositionBanner();
				}
````
lets name is like journeys_utils._centerOverlay();

Reply:
nice, good suggestion, done